service/history/configs/config.go

```diff
@@ -361,8 +363,10 @@ func NewConfig(
 		HistoryShardLevelCacheMaxSize:         dc.GetIntProperty(dynamicconfig.HistoryCacheShardLevelMaxSize, 512),
 		EnableAPIGetCurrentRunIDLock:          dc.GetBoolProperty(dynamicconfig.EnableAPIGetCurrentRunIDLock, false),
 
-		EventsCacheMaxSizeBytes: dc.GetIntProperty(dynamicconfig.EventsCacheMaxSizeBytes, 512*1024), // 512KB
-		EventsCacheTTL:          dc.GetDurationProperty(dynamicconfig.EventsCacheTTL, time.Hour),
+		EventsCacheMaxSizeBytes:           dc.GetIntProperty(dynamicconfig.EventsCacheMaxSizeBytes, 512*1024),               // 512KB
+		EventsShardLevelCacheMaxSizeBytes: dc.GetIntProperty(dynamicconfig.EventsShardLevelCacheMaxSizeBytes, 512*512*1024), // 256MB
```

Comment:
I think you mean the host level one is 256MB.

Reply:
Yes. fixed it.